Wikipedia/Categories/NSProcessInfo+WMFOperatingSystemVersionChecks.h

```diff
@@ -0,0 +1,25 @@
+//
+//  NSProcessInfo+WMFOperatingSystemVersionChecks.h
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 11/11/15.
+//  Copyright © 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSProcessInfo (WMFOperatingSystemVersionChecks)
+
+/**
+ *  @return Whether or not the current OS version is less than 9.0.0.
+ *
+ *  @note This method is preferred to the parameterized ones, since it will automatically mark code as deprecated once
+ *        the deployment target is raised.
+ */
+- (BOOL)wmf_isOperatingSystemVersionLessThan9_0_0 WMF_DEPRECATED_WHEN_DEPLOY_AT_LEAST_9;
+
+- (BOOL)wmf_isOperationSystemMajorVersionAtLeast:(NSUInteger)version;
```

Comment:
Well for me, I think it is pretty convenient to not have to make a struct every time I want to check the version. I had many functions / macros that did that in the past just haven't added them here.